_providers/mail2tor.md

```diff
@@ -0,0 +1,23 @@
+---
+name: mail2tor
+status: PREPARATION
+domains:
+- mail2tor.com
+strict_tls: true
+server:
+- type: imap
+  hostname: g77kjrad6bafzzyldqvffq6kxlsgphcygptxhnn4xlnktfgaqshilmyd.onion
+  port: 143
+  socket: PLAIN
+- type: smtp
+  hostname: xc7tgk2c5onxni2wsy76jslfsitxjbbptejnqhw6gy2ft7khpevhc7ad.onion
+  port: 25
+  socket: PLAIN
+before_login_hint: |
+  Tor is needed to connect to the email servers.
+website: http://mail2torjgmxgexntbrmhvgluavhj7ouul5yar6ylbvjkxwqf6ixkwyd.onion/
+# website: http://mail2tor.com/
+last_checked: 2021-08-15
+---
+
+Email servers are only reachable over [Tor](https://www.torproject.org/).
```

Comment:
Is there some information on configuration needed? Like "socks5 proxy at localhost:9050", or is it auto-configured if it sees a .onion domain?

Reply:
You need to configure socks proxy to localhost:9050, but afaik if you use Tor browser instead of the standalone Tor, then the port is different.

How to configure Tor is out of scope of the provider database, you can even use https://www.torbox.ch/ and avoid configuration of your client device completely.

Reply:
Alright :) then let's merge it as it is, and at some point we can link to a blogpost which talks about configuration?

Reply:
Is it possible to not list it on providers.delta.chat? It's mainly for conveniance of testing mail2tor addresses in my opinion, until socks5 support actually landed in a released version.

Reply:
It requires tweaking index.html and adding a new attribute like "hidden: true" to .yaml.

I don't think it's a problem for mail2tor that it's listed, it says clearly that you need to somehow use the Tor. If you can't, then you can't use the provider.

Reply:
yip, i also think, listing is okay. however, for whatever reason, the preview seems to be broken and does not pick up the status `PREPARATION` and does not show the hints therefore. might be a preview-only issue, however.

Reply:
The correct preview link is https://testrun.org/provider/2694122fdab5ac5094d212da3c7957ed9475f43a/_site/ - there it works.

The github actions create a new preview per commit instead of branch/PR... a bit weird behvaior, in most other repositories we solved this in a more clever way.

Just the comment providing the link is affected, if you access the preview from the checks below, it works - this one is updated.

Reply:
thanks for explanations. indeed, this is confusing. if it is not easily possible, to get a stable preview link, it would be better if the github-action post-bot says:

"Check out the page preview at: Show all checks / Page Preview / Details" 

instead of

"Check out the page preview at https://testrun.org/provider/1234567890...unstable-id/_site/"

Reply:
i filed an issue for that at https://github.com/deltachat/provider-db/issues/197